docker/example_run.sh

```diff
@@ -5,16 +5,16 @@
 # https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/latest/install-guide.html
 
 # Set the following variables:
-IMAGE=
+IMAGE=lmcache/vllm-openai:2025-05-17-v1 # WARNING: Cannot use latest or omit the tag!!!
```

Comment:
Lets not set that here. Want the image to be specified. Also, the `latest` tag seems to be working (see https://github.com/LMCache/LMCache/issues/686#issuecomment-2893607922).